src/alchemlyb/preprocessing/subsampling.py

```diff
@@ -102,101 +290,208 @@ def statistical_inefficiency(df, series=None, lower=None, upper=None, step=None,
     decreases a false sense of accuracy and is deemed the more careful and
     conservative approach.
 
+    References
+    ----------
+    [1] J. D. Chodera, W. C. Swope, J. W. Pitera, C. Seok, and K. A. Dill. Use of the weighted
+        histogram analysis method for the analysis of simulated and parallel tempering simulations.
+        JCTC 3(1):26-41, 2007.
+
     See Also
     --------
     pymbar.timeseries.statisticalInefficiency : detailed background
     pymbar.timeseries.subsampleCorrelatedData : used for subsampling
 
 
+    .. versionchanged:: 0.4.0
+       The ``series`` keyword was replaced with the ``column`` keyword.
+       The function no longer takes an arbitrary series as input for
+       calculating statistical inefficiency.
+
     .. versionchanged:: 0.2.0
        The ``conservative`` keyword was added and the method is now using
-       ``pymbar.timeseries.statisticalInefficiency()``; previously, the statistical
-       inefficiency was _rounded_ (instead of ``ceil()``) and thus one could
-       end up with correlated data.
+       ``pymbar.timeseries.subsampleCorrelatedData()``; previously, the
```

Comment:
Did pymbar change the function name or was this a mistake?

Reply:
This was a mistake in the docstring that I fixed here; the same mistake was not made in the equivalent line for `equilibrium_detection`.